suites/apis/clientCredGrantTest.js

```diff
@@ -117,8 +117,13 @@ Scenario('Client Credentials Grant Type interaction with Requestor @clientCreds'
   const requestStatusSigned = await requestorTasks.getRequestStatus(I.cache.requestID);
   console.log(`Status of the request is:${requestStatusSigned}`);
 
-  // do not uncomment this code below until PXP - 10229 is done
   // getting the list of users access request
-  // const list = await requestorTasks.getRequestList(clientAccessToken);
-  // console.log(list);
+  const list = await requestorTasks.getRequestList(clientAccessToken);
+  if (process.env.DEBUG === 'true') {
+    console.log(list.data);
+  }
+  if (list.data.length > 0) {
+    list.data.forEach(async (obj) => {
+      expect(obj).to.have.property('request_id');
+    })}
```

Comment:
i meant that we should check that one of the returned requests has `request_id == I.cache.requestID` 🙂

Reply:
other than that lgtm